README.md

```diff
@@ -30,6 +39,8 @@ In your terminal, run:
 docker-compose up backend frontend
 ```
 
+or simply `make local`.
```

Comment:
How does a contributor know whether they should use Docker or Make?

Reply:
@philipdickinson let's keep make for internal use, but expose docker commands for public use. Thoughts?